vivisect/symboliks/common.py

```diff
@@ -235,6 +235,27 @@ def update(self, emu):
 
     def setSymKid(self, idx, kid):
         '''
+        Creates or sets a child for the current symbolik object
+
+        The sticky bit here is when the child already exists and we're setting it to a new value
+        (as in the case of things like symbolik reduction). In that case we have to invalidate
+        the symcaches of every parent node up all of the ASTs that the child exists in. However,
+        that can get exceedingly repetative and expensive, especially in the reduction cases
+        where we don't need to constantly clear the ASTs, since during the traversal they'll be
+        cleared once and then never need to cleared again. And if they are populated for whatever
```

Comment:
never need to be cleared again (because ...)
because of how and when the cache is created...  right?